src/applications/disability-benefits/all-claims/content/privateMedicalRecords.jsx

```diff
@@ -1,41 +1,84 @@
 import { ANALYTICS_EVENTS, HELP_TEXT_CLICKED_EVENT } from '../constants';
 
 const {
   openedPrivateRecordsAcknowledgment,
-  openedPrivateChoiceHelp,
+  // openedPrivateChoiceHelp,
 } = ANALYTICS_EVENTS;
 
 export const privateRecordsChoiceHelp = (
-  <div className="private-records-choice-help">
-    <VaAdditionalInfo
-      trigger="Which should I choose?"
-      disableAnalytics
-      onClick={() =>
-        recordEventOnce(openedPrivateChoiceHelp, HELP_TEXT_CLICKED_EVENT)
+  <p>
+    <va-accordion open-single>
+      <va-accordion-item
+        bordered
+        header="What happens when you upload your non-VA treatment records yourself"
+        id="get-records-yourself"
+      >
+        <p>
+          We can review your claim more quickly if you upload your records
+          yourself. You can upload .pdf, .jpeg, or .png files.
+        </p>
+      </va-accordion-item>
+    </va-accordion>
+
+    <va-accordion open-single>
+      <va-accordion-item
+        bordered
+        header="What happens when you ask us to get your non-VA treatment records"
+        id="have-va-get-your-records"
+      >
+        <p>
+          We’ll ask you to give us information about the provider or hospital
+          that treated you. We’ll use this information to get your records. It
+          may take some time for us to get them. This means that it could take
+          longer for us to decide your claim. We’ll try at least 2 times to get
+          your records from the provider or hospital. If we can’t get them,
+          we’ll contact you to let you know and tell you if there’s anything you
+          need to do.
+        </p>
+      </va-accordion-item>
+    </va-accordion>
+  </p>
+);
+
+// Banner component that clears the session flag on mount
+const RecordsConfirmAlertBanner = () => {
+  const bannerElement = useRef(null);
+  React.useEffect(() => {
+    try {
+      if (typeof window !== 'undefined') {
+        window.sessionStorage.removeItem('needsShownNew4142Alert');
       }
+    } catch (e) {
+      // noop
+    }
+    focusElement(bannerElement.current);
+  }, []);
+
+  return (
+    <va-banner
+      data-label="Info banner"
+      headline="Confirm how to provide your non-VA records"
+      type="warning"
+      visible
     >
-      <h3 className="vads-u-font-size--h4">You upload your medical records</h3>
       <p>
-        If you upload a digital copy of all your medical records, we can review
-        your claim more quickly. Uploading a digital file works best if you have
-        a computer with a fast Internet connection. The digital file can be
-        uploaded as a .pdf or other photo file format, like a .jpeg or .png.
+        You previously chose not to upload your private medical records and gave
+        us permission to get them from your provider. But we updated our terms
+        and conditions for the release of these records.
+        <br />
+        If you still want us to get them, select <b>No</b> and review the
```

Comment:
If the **No** should be emphasized by screen readers, it should be styled with `<strong>` tags. Otherwise, if the intent is purely presentational, <b> will suffice.